doc/SPIRE101.md

```diff
@@ -38,35 +38,27 @@ Note: If you don't already have Docker installed, please follow these [installat
 
 ## Walkthrough
 
-1.  Start the SPIRE container using docker-compose.
+1.  Build the development Docker image.
 
-        docker-compose up -d
+        make dev-image
 
-2.  Check to make sure the spire-dev docker container is running.
+2.  Run a shell in the development Docker container.
```

Comment:
Are these first two steps even necessary?  `make build` will download the necessary toolchain required to build the binaries...

Reply:
I wanted to keep this walkthrough using a dev container, so you can add a user to play with workload attestation without altering your environment. What do you think?

Reply:
sgtm!